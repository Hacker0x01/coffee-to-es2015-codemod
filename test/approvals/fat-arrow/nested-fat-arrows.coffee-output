var handleChange;

handleChange = (function(_this) {
  return function(event) {
    var nested;
    event.preventDefault();
    return nested = function() {
      var superNested;
      return superNested = function() {
        return console.info(_this);
      };
    };
  };
})(this);
