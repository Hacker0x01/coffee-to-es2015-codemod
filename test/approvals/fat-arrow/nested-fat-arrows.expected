var handleChange;

handleChange = event => {
  var nested;
  event.preventDefault();
  return nested = () => {
    var superNested;
    return superNested = () => {
      return console.info(this);
    };
  };
};
